manifests/install/rabbitmqadmin.pp

```diff
@@ -28,6 +28,8 @@
     $default_user = $rabbitmq::default_user
     $default_pass = $rabbitmq::default_pass
     $management_ip_address = $rabbitmq::management_ip_address
+    $extra_opts = $rabbitmq::archive_options
```

Comment:
Here, I would use the convention of keeping the same param name,
`archive_options = $rabbitmq::archive_options` (and then using that below).

Reply:
shall do